sechub-sereco/src/main/java/com/daimler/sechub/sereco/importer/SarifV1JSONImporter.java

```diff
@@ -0,0 +1,181 @@
+package com.daimler.sechub.sereco.importer;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+import com.contrastsecurity.sarif.SarifSchema210;
+import com.contrastsecurity.sarif.ThreadFlow;
+import com.contrastsecurity.sarif.CodeFlow;
+import com.contrastsecurity.sarif.Location;
+import com.contrastsecurity.sarif.PhysicalLocation;
+import com.contrastsecurity.sarif.ReportingDescriptorReference;
+import com.contrastsecurity.sarif.Result;
+import com.contrastsecurity.sarif.Result.Level;
+import com.daimler.sechub.commons.model.ScanType;
+import com.daimler.sechub.sereco.metadata.SerecoCodeCallStackElement;
+import com.daimler.sechub.sereco.metadata.SerecoMetaData;
+import com.daimler.sechub.sereco.metadata.SerecoSeverity;
+import com.daimler.sechub.sereco.metadata.SerecoVulnerability;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Component
+public class SarifV1JSONImporter extends AbstractProductResultImporter {
```

Comment:
Please rename the file and class from "SarifV1JSONImporter.java" to "SarifJSONImporterV1.java". The file name is ambiguous, because starting with "SarifV1" could mean the "JSON Importer" supports only "SARIF version 1" or the "JSON Importer" is in version one. Moving the version to the end clarifies, that the "JSON Importer" is in version 1.

Reply:
![image](https://user-images.githubusercontent.com/71753812/109132835-8e1aa080-7754-11eb-896c-5f7b514a762c.png)


This is matching the naming pattern of importers. I think, the V1 is because of the SecHub API V1.

I would like to avoid refactoring of all this classes and their appearances.

Reply:
In this case, please add a comment explaining which SARIF versions are supported with a link to the SARIF standard. For example: https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html